ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviderForm.tsx

```diff
@@ -450,6 +451,49 @@ function AuthProviderForm({
                             </p>
                         </Alert>
                     </div>
+                    {selectedAuthProvider.requiredAttributes && (
```

Comment:
I wonder if we should allow for multiple required attributes instead of one. There is no use for multiple required attributes right now, and UI can be simplified(both code and how it looks)

Reply:
Whatever we do, our UI should be in sync with the backend.

Reply:
I don't really see the overhead of having multiple required attributes right now. I also don't see how much code it would save on the FE side, as the benefit would be to not have a map call.
If we anyways later on potentially have the goal of providing this with users, I don't see a big overhead for keeping multiple required ones for now and I'm not sure it would significantly simplify things FE / BE wise.

Reply:
I think the important point is how it looks. 

For now, our only use case is org_id check(single attribute) and creating an entirely new section might be a little bit of overkill. 
I would be curious to hear @vjwilson opinion on the topic. Maybe, we can unite this section with the one above or below(and potentially generalize the name)?

Reply:
I agree that keeping the design is better than involving UX 👍